app/src/main/res/drawable/other_tracker_privacy_dashboard_bg.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+  ~ Copyright (c) 2020 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+    <solid android:color="?attr/toolbarIconColor" />
+    <size
+        android:height="24dp"
```

Comment:
just wanted to check if this is 24dp or it should be 22dp like the other privacy grades

Reply:
This one should be 24dp because is for the privacy dashboard where the icons are 24dp rather than 22dp.